package.json

```diff
@@ -65,6 +65,7 @@
     "read-chunk": "3.2.0",
     "semver": "7.3.5",
     "socket.io": "4.4.1",
+    "tiny-typed-emitter": "2.1.0",
```

Comment:
This seems like a fairly small library. We try and keep our dependencies to a minimum; is there a way around this? Can we manually type just the features we use?

Reply:
Typed just `on` and `emit` for now.